lib/pengine/status.c

```diff
@@ -98,23 +99,25 @@ check_for_deprecated_rules(pcmk_scheduler_t *scheduler)
  *  - A list of nodes that need to be stonith'd
  *  - A list of nodes that need to be shutdown
  *  - A list of the possible stop/start actions (without dependencies)
+ *
+ * \return Standard Pacemaker return code
  */
-gboolean
-cluster_status(pcmk_scheduler_t * scheduler)
+int
+pcmk_unpack_scheduler_input(pcmk_scheduler_t *scheduler)
 {
     const char *new_version = NULL;
     xmlNode *section = NULL;
 
     if ((scheduler == NULL) || (scheduler->input == NULL)) {
-        return FALSE;
+        return EINVAL;
```

Comment:
I'm thinking `CRM_CHECK()` (otherwise we should log an error for consistency with the other return)